layout/navigation.html

```diff
@@ -18,11 +18,97 @@
   <h1>Primary Navigation<span class="status dev-ready">Complete</span></h1>
   <p>The primary navigation is used for the main sections of the application or for dropdown categories that organize sections into higher level groups.</p>
 
+  <div class="row" id="side-nav">
+    <div class="col-md-4">
+      <h2>Left Navigation</h2>
+      <p>The Left Navigation component is used on all web applications that desire to implement a highly compliant version of the Helix Design Language.</p>
+      <ul>
+        <li>Can extend to fit any navigation IA via scrolling when the list of navigational items exceeds the screen view-height.</li>
+        <li>Preferred implementation includes only primary and secondary navigational links.</li>
+          <ul>Note: A less preferred implementation that includes <a href="{{page.url}}#tertiary"> tertiary navigational</a> elements is also provided.</ul>
```

Comment:
:+1: I probably have a set of styles squirreled away from the IX sites for that sort of note that we can use...